backend/src/controllers/items-controller.ts

```diff
@@ -35,7 +36,8 @@ export const createItem = async (req: CustomRequest, res: Response) => {
         description: description,
         pickupLocation: pickupLocation,
         pickupTime: pickupTime,
-        itemOwnerId: req.token.userId
+        itemOwnerId: req.token.userId,
+        groupId: req.body.groupId
```

Comment:
Before adding the `groupId` to the `Item`, check that a `Group` object exists with its `_id` equal to `groupId`